### PR TITLE
{lang}[fosscuda/2019b] rstudio v1.2.5033

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

Alternative using Java 8: https://github.com/easybuilders/easybuild-easyconfigs/pull/9791

### CODE REVIEW

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
taurusi6197.taurus.hrsk.tu-dresden.de - Linux RHEL 7.4, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c82822dbac3cc78e2b59ca58457d8f0f for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
taurusml29 - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/72446157e308867a913359fda49571fa for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**FAILED**
Build succeeded for 19 out of 23 (1 easyconfigs in this PR)
taurusml29 - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/b3ad476ea82703afa5158d515dcfdbef for a full test report.

File: easybuild/easyconfigs/r/rstudio/rstudio-1.2.5033_update_gwt.patch [hunk 1]
```diff
@@ -0,0 +1,22 @@
+--- a/dependencies/common/install-gwt	2020-02-04 11:16:49.907923640 +0000
```
Comment (lexming) [inline easybuild/easyconfigs/r/rstudio/rstudio-1.2.5033_update_gwt.patch:1]: Please add a description and authorship of this patch.

Comment (Flamefire) [reply easybuild/easyconfigs/r/rstudio/rstudio-1.2.5033_update_gwt.patch:4]: Done: https://github.com/easybuilders/easybuild-easyconfigs/pull/9826/commits/06364476f53f237afd71f77501fc11b6d6ed4c63

Note: Patch originally extracted from https://github.com/bear-rsg/easybuild-easyconfigs/pull/52 and suggested by @edmondac

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node254.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2683 v4 @ 2.10GHz, Python 2.7.5
See https://gist.github.com/8a9549087f1bf45663bf53678615f53b for a full test report.

**Comment by lexming** [comment]
Going in, thanks @Flamefire !

**Pull request merged**